ansible/roles/web/tasks/github-markup.yml

```diff
@@ -0,0 +1,34 @@
+---
+# github-markup tool
```

Comment:
Although I don't like having any ruby dependencies, I also think that this is for the best and will allow us to be consistent with the way Github renders the files.